rust/lance-index/src/vector/pq.rs

```diff
@@ -113,7 +114,7 @@ impl ProductQuantizer {
         let sub_dim = dim / num_sub_vectors;
         let values = flatten_data
             .values()
-            .par_chunks(dim)
+            .chunks_exact(dim)
```

Comment:
🔪

Reply:
Curious about the backstory here. Isn't the the place where we brought in `rayon`?

Reply:
I don't believe we did any benchmarking when it was brought in.  Also, it was brought in with another change (removing async calls) that may indeed have had a benefit to perf.